implementations/micrometer-registry-stackdriver/src/main/java/io/micrometer/stackdriver/StackdriverMeterRegistry.java

```diff
@@ -338,8 +344,13 @@ public Builder metricServiceSettings(Callable<MetricServiceSettings> metricServi
             return this;
         }
 
+        Builder clientFactory(MetricServiceClientFactory clientFactory) {
```

Comment:
We usually leave a comment such as `// VisibleForTesting` to note why some API is not private but also not public.